content/enterprise/operational-modes/index.mdx

```diff
@@ -0,0 +1,162 @@
+---
+page_title: Operational Modes - Terraform Enterprise
+description: >-
+  Learn about the different operational modes of Terraform Enterprise.
+---
+
+# Terraform Enterprise Operational Modes
+
+Before deploying Terraform Enterprise, you must choose an operational mode.
+Each operational mode changes where Terraform Enterprise stores its data.
+
+## Terraform Enterprise Data
+
+Terraform Enterprise data utilizes the following types of data:
+
+- **PostgreSQL Database** - Stateful Terraform Enterprise application data. (e.g. workspace settings, organization settings, run information, user information)
+
+- **Object Storage** - Artifacts generated by Terraform Enterprise. (e.g. state files, plan files, run logs, configuration versions)
+
+- **Vault** - Encryption keys used to encrypt and decrypt objects within object storage.
+
+- **Configuration** - Configuration settings. (e.g. the hostname, object storage credentials, database credentials, concurrency settings, etc.)
```

Comment:
```suggestion
- **PostgreSQL Database:** - Stateful Terraform Enterprise application data. This includes workspace settings, organization settings, run information, and user information.

- **Object Storage:** Artifacts that Terraform Enterprise produces during operations. This includes state files, plan files, run logs, configuration versions.

- **Vault:** Encryption keys that encrypt and decrypt objects within object storage.

- **Configuration:** Configuration settings. This includes the hostname, object storage credentials, database credentials, concurrency settings, etc.
```

Reply:
should we also link to the PostgreSQL requirements page? https://www.terraform.io/enterprise/requirements/data-storage/postgres-requirements

Reply:
I don't think so. We're just listing the types of data here.